CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [5.0.0] - 2024-04-15
+
+This is a complete overhaul of the plugin, based on extensive feedback and testing in screen
+readers. Some breaking changes have been made to improve the defaults. If you haven't customized the
+plugin options, it's safe to upgrade, otherwise consult the readme.
+
+- Rethink announcements: use assertive live region and announce after a delay (@ScoobyDid)
+- Focus `body` on navigation instead of `main`
+- Only use `h1` for announcements, ignore `h2`
```

Comment:
By default, only use the `h1` for announcements, ...?

Reply:
👌👌